apps/communications/contacts/style/contacts.css

```diff
@@ -211,6 +211,11 @@ section[role="region"] > gaia-header .icon.icon-edit-contact {
   width: 30rem;
 }
 
+#contact-form input {
```

Comment:
You should modify this instead:
`input::-moz-placeholder, textarea::-moz-placeholder {`
`overflow: hidden;`
`text-overflow: ellipsis;`
`}`

We want to modify the styles of the placeholder, not the styles of the input! :)